guides/common/modules/proc_exporting-a-repository.adoc

```diff
@@ -24,59 +24,28 @@ The export contains the following data:
 
 You need all the files, `tar.gz`, `toc.json` and `metadata.json`, to be able to import.
 
-.Prerequisites
-
-To export the contents of a repository, ensure that {ProjectServer} from which you want to export, meets the following conditions:
-
+.Prerequisites to exporting a repository on {ProjectServer}
 * Ensure that the export directory has enough free storage space to accommodate the export.
 * Ensure that the `/var/lib/pulp/exports` directory has enough free storage space equivalent to the size of the repositories being exported for temporary files created during the export process.
 * Ensure that you set download policy to *Immediate* for the repository within the Library lifecycle environment you export.
 For more information, see xref:Download_Policies_Overview_{context}[].
 * Ensure that you synchronize Products that you export to the required date.
 
-.Export a Repository
-
-. Use the repository name or ID to export.
+.Procedure
+. Export a repository using the product name and repository name:
 +
 [options="nowrap" subs="+quotes"]
 ----
 # hammer content-export complete repository \
- --organization="_My_Organization_" \
- --product="_My_Product_" \
- --name="_My_Repository_"
-----
-
-. Optional: Verify that the exported archive is located in the export directory:
-+
-[options="nowrap" subs="+quotes"]
+--name="_My_Repository_" \
+--product="_My_Product_"
 ----
-# ls -lh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00
-total 68M
--rw-r--r--. 1 pulp pulp 68M Mar  2 03:35 export-1e25417c-6d09-49d4-b9a5-23df4db3d52a-20210302_0335.tar.gz
--rw-r--r--. 1 pulp pulp 333 Mar  2 03:35 export-1e25417c-6d09-49d4-b9a5-23df4db3d52a-20210302_0335-toc.json
--rw-r--r--. 1 root root 443 Mar  2 03:35 metadata.json
-----
-
-.Export a Repository with Chunking
-
-In many cases the exported content archive may be several gigabytes in size.
-If you want to split it into chunks of smaller size, you can use the `--chunk-size-gb` argument in the export command and limit the size by an integer value in gigabytes.
-
-. Export content into archive chunks of a limited size, such as 2 GB:
 +
-[options="nowrap" subs="+quotes"]
-----
-# hammer content-export complete repository \
---chunk-size-gb=_2_ \
---organization="_My_Organization_" \
---product="_My_Product_" \
---name="_My_Repository_"
-
-Generated /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00/metadata.json
-----
-. Optional: View the exported data:
+The size of the exported archive depends on the number and size of the packages within the repository.
+If you want to split the exported archive into chunks, export your repository using the `--chunk-size-gb` argument to limit the size by an integer value in gigabytes, for example `---chunk-size-gb=_2_`.
```

Comment:
```suggestion
+
[NOTE]
====
The size of the exported archive depends on the number and size of the packages within the repository.
If you want to split the exported archive into chunks, export your repository using the `--chunk-size-gb` argument to limit the size by an integer value in gigabytes, for example `---chunk-size-gb=_2_`.
====
```
Maybe this could be a note?

Reply:
Applied.